### ISSUE TITLE
Instruções do processo de trabalho

### ISSUE THREAD

**Issue by Abreums** [gp-ti/timwood#4]
O resultado deverá ser a criação de páginas na wiki com informações sobre os desperdícios em projetos e ações para eliminá-los.
Porém, para criar uma página é preciso seguir um processo:
1) Uma ISSUE deve ser criada.
2) A ISSUE deve ter a colaboração de mais de 3 envolvidos no amadurecimento da idéia. 
3) Uma tarefa no projeto deverá ser criada a partir da ISSUE.
4) Qualquer pessoa pode solicitar executar a tarefa
5) Uma pessoa não poderá ter mais de uma tarefa no estado in-progress 
6) Ao terminar de resumir as informações na WIKI o responsável move o cartão para "ready-to-review"
7) Outro recurso deverá fazer a revisão e mover o cartão para done.

**Comment by Ivanilson6** [comment]
Professor isso deve ser feito aqui no guitHub?

**Comment by Abreums** [comment]
sim. Atenção: a pessoa que criar a tarefa, não poderá executá-la. Ela deverá contar com a colaboração de outra para executar a tarefa!
